<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGLDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyGL" name="mygl">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>618</width>
      <height>432</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="List_Vertex">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>141</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="List_Edge">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>141</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="List_Face">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>20</y>
      <width>141</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Update_button">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Subdivide_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>340</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Subdivide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bind_Skin">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bind Skin</string>
    </property>
   </widget>
   <widget class="QListWidget" name="List_Joints">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>20</y>
      <width>141</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="X_plus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="X_decrease">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Y_plus">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Y+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Y_decrease">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Y-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Z_plus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Z+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Z_decrease">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Z-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCamera_Controls"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCamera_Controls">
   <property name="text">
    <string>Camera Controls</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGL</class>
   <extends>QOpenGLWidget</extends>
   <header>mygl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
